src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedEmbeddedExtensionMarkerNameAttributeSymbol.cs

```diff
@@ -39,5 +48,168 @@ internal override AttributeUsageInfo GetAttributeUsageInfo()
+            public override RefKind RefKind => RefKind.None;
+            public override ImmutableArray<CustomModifier> RefCustomModifiers => [];
+            public override MethodSymbol GetMethod { get; }
+            public override MethodSymbol? SetMethod => null;
+            public override Symbol ContainingSymbol => _backingField.ContainingSymbol;
+            public override Accessibility DeclaredAccessibility => Accessibility.Internal;
+
+            public override ImmutableArray<Location> Locations => [];
+            public override ImmutableArray<SyntaxReference> DeclaringSyntaxReferences => [];
+            public override ImmutableArray<PropertySymbol> ExplicitInterfaceImplementations => [];
+            public override ImmutableArray<ParameterSymbol> Parameters => [];
+            public override bool IsIndexer => false;
+            public override bool IsStatic => false;
+            public override bool IsVirtual => false;
+            public override bool IsOverride => false;
+            public override bool IsAbstract => false;
+            public override bool IsSealed => false;
+            public override bool IsExtern => false;
+            internal override bool IsRequired => false;
+            internal override bool HasSpecialName => false;
+            internal override CallingConvention CallingConvention => CallingConvention.HasThis;
+            internal override bool MustCallMethodsDirectly => false;
+            internal override bool HasUnscopedRefAttribute => false;
+            internal override ObsoleteAttributeData? ObsoleteAttributeData => null;
+            internal override int TryGetOverloadResolutionPriority() => 0;
+        }
+
+        private sealed partial class NameGetAccessorMethodSymbol : SynthesizedMethodSymbol
+        {
+            private readonly NamePropertySymbol _nameProperty;
+            private readonly SynthesizedFieldSymbol _backingField;
+
+            public NameGetAccessorMethodSymbol(NamePropertySymbol nameProperty, SynthesizedFieldSymbol backingField)
+            {
+                _nameProperty = nameProperty;
+                _backingField = backingField;
+            }
+
+            public override string Name => "get_Name";
+            internal override bool HasSpecialName => true;
+            public override MethodKind MethodKind => MethodKind.PropertyGet;
+            public override Symbol AssociatedSymbol => _nameProperty;
+            public override Symbol ContainingSymbol => _nameProperty.ContainingSymbol;
+
+            internal override bool SynthesizesLoweredBoundBody => true;
+
+            internal override void GenerateMethodBody(TypeCompilationState compilationState, BindingDiagnosticBag diagnostics)
+            {
+                SyntheticBoundNodeFactory F = new SyntheticBoundNodeFactory(this, this.GetNonNullSyntaxNode(), compilationState, diagnostics);
```

Comment:
>this.GetNonNullSyntaxNode() [](http://example.com/codeflow?start=82&length=27)

Is there a chance that is is going to produce non-Dummy node? If not, consider using it directly.